BMM.UI.Android/Application/Activities/MainActivity.cs

```diff
@@ -37,24 +33,19 @@ namespace BMM.UI.Droid.Application.Activities
          ScreenOrientation = ScreenOrientation.Portrait,
          Exported = true
     )]
-    [IntentFilter(
```

Comment:
doesn't removing this cause problems if the app is already opened and someone clicks a link?

Reply:
I have checked it on a few devices and it's working as expected.

Reply:
I finally decided to leave it on MainActivity, bacause I found a different solution for showing a splashscreen, when opening the application from deep link. We are sure that's working properly on MainActivity, so there is no need to introduce some risk that it will stop working in some cases.